### DISCUSSION TITLE
Download torrent files

### DISCUSSION THREAD

**Discussion by Radiomaka** [anasty17/mirror-leech-telegram-bot#1377] [Q&A]
HI! Thank you for this adorable bot
Few days ago i updated bot, because youtube links was not downloading. But now there is another problem
How can i start torrent download now? In previous version it was enough to reply on uploaded torrent file by "/l".
Now, on every message i get
" /cmd link |newname pswd: xx(zip/unzip)
By replying to link/file:
/cmd |newname pswd: xx(zip/unzip).........."
There is no problems in downloading direct links

**Comment by anasty17** [comment]
There no problem in code. Update ur code and promote the bot!

**Reply by anasty17** [reply]
Ur bot is not admin mostly

**Reply by Radiomaka** [reply]
It is up to date. Just yesterday reinstalled from 0 on  new VPS.

**Reply by anasty17** [reply]
Just add UPSTREAM_REPO to stay up to date and promote ur bot to admin

**Reply by ujwalnayak3** [reply]
sir i didnt understand How to prepare a bot

**Comment by Radiomaka** [comment]
Added UPSTREAM_repo in configHow i can promote bot to admin?
    понедельник, 20 февраля 2023 г., 19:44:19 GMT+3, Anas ***@***.***> написал(-а):  
 
 


Just add UPSTREAM_REPO to stay up to date and promote ur bot to admin

**Comment by anasty17** [comment]
Join the group and i will tell u

**Comment by adirane45** [comment]
> HI! Thank you for this adorable bot Few days ago i updated bot, because youtube links was not downloading. But now there is another problem How can i start torrent download now? In previous version it was enough to reply on uploaded torrent file by "/l". Now, on every message i get " /cmd link |newname pswd: xx(zip/unzip) By replying to link/file: /cmd |newname pswd: xx(zip/unzip).........." There is no problems in downloading direct links


Just check u r cmd in my case it is /ql for leech and /qm for mirror
so to use torrent file upload it on bot reply it and use **/ql -s**
<img width="632" height="792" alt="image" src="https://github.com/user-attachments/assets/0f6a53f2-70ca-4e1c-812a-d2ddca3ae813" />
